docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/webpack-aliases-removed.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Webpack Aliases are removed
+description: A simplified approach of aliasing in Strapi v5. 
+sidebar_label: Webpack Aliases removed
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - dependencies
+ - webpack
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Webpack Aliases are removed
+
+In Strapi v5, webpack aliases are removed ensuring better compatibility and reduced dependency conflicts. 
+
+<Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+- We maintained a specific list of dependencies that were aliased in webpack configuration. This ensured that plugins consistently used our versions of certain libraries like the design-system. <br/> <br/> However, this approach led to many bugs.
```

Comment:
```suggestion
- We maintained a specific list of dependencies that were aliased in webpack configuration. This ensured that plugins consistently used our versions of certain libraries like the design-system.
```

I would remove this. No need to shoot ourselves in the foot 😂

Reply:
Okay cool